### DISCUSSION TITLE
[Question] What's a good delay on a VPS close to Discord's datacenter?

### DISCUSSION THREAD

**Discussion by voidpls** [Vedza/NitroSniperGo#242] [General]
It'd be nice to know the delays of some other people, along with their hosting situation, for a benchmark to optimize our own delays.

**Comment by voidpls** [comment]
\+ The lowest latency should be on a GCP VPS, as Discord uses GCP for their API. 

Do we know which GCP locations Discord uses?

**Reply by Vedza** [reply]
It depends on the guilds
